test/framework/integration_framework/integration_test_framework.cpp

```diff
@@ -154,6 +156,8 @@ namespace integration_framework {
                                             block_store_path,
                                             kLocalHost,
                                             torii_port_,
+                                            0,
```

Comment:
Why you are not using here the value of previously defined constant ``constexpr size_t kDefaultToriiTlsPort = 0;``?
I believe that will make the code more clear.

Reply:
Fixed in latest version